cvat/settings/base.py

```diff
@@ -667,3 +674,7 @@ class CVAT_QUEUES(Enum):
 }
 
 BUCKET_CONTENT_MAX_PAGE_SIZE =  500
+
+IMPORT_CACHE_FAILED_TTL = timedelta(days=90)
```

Comment:
Please comment why such a long time is needed. Maybe, it should only be done on the platform, while the opensource version can have lesser time?

Reply:
The [default](https://python-rq.org/docs/) value is 1 year, I guess that we need to set some optimal value to not keep it too long, but still be able to see the details of failed job